mesh_client/__init__.py

```diff
@@ -337,59 +409,66 @@ def send_message(
         compression for sending is enabled as a constructor option.
```

Comment:
i think this doc string needs updating, removing the partner_id, and smtp headers

Reply:
partner_id isn't going .. it's process_id that's been removed ..  (but yes .. that was left behind in docstrings!)